nicintel_eeprom.c

```diff
@@ -275,6 +409,25 @@ static const struct opaque_master opaque_master_nicintel_ee = {
 	.erase = nicintel_ee_erase,
 };
 
+static int nicintel_ee_shutdown_i210(void *arg)
```

Comment:
We need to define the dummy argument, the function is passed to register_shutdown with this prototype

`flash.h:int register_shutdown(int (*function) (void *data), void *data);`

other drivers do the same, you may want to take a look to dediprog_shutdown foe example.